core/src/main/scala/com/github/mrpowers/spark/fast/tests/SchemaComparer.scala

```diff
@@ -55,24 +57,32 @@ object SchemaComparer {
         false
       } else {
         val zipStruct = if (ignoreColumnOrder) s1.sortBy(_.name) zip s2.sortBy(_.name) else s1 zip s2
-        zipStruct.forall { t =>
-          (t._1.nullable == t._2.nullable || ignoreNullable) &&
-          (t._1.name == t._2.name || ignoreColumnNames) &&
-          equals(t._1.dataType, t._2.dataType, ignoreNullable, ignoreColumnNames, ignoreColumnOrder)
+        zipStruct.forall { case (f1, f2) =>
+          (f1.nullable == f1.nullable || ignoreNullable) &&
```

Comment:
Interesting that this slight mistake doesn't cause a test failure. I will add test case

Reply:
Will be addressed in https://github.com/mrpowers-io/spark-fast-tests/pull/167